test/extended/builds/cluster_config.go

```diff
@@ -373,11 +336,6 @@ var _ = g.Describe("[Feature:Builds][Serial][Slow][Disruptive] alter builds via
 				o.Expect(err).NotTo(o.HaveOccurred())
 				checkOCMProgressing(operatorv1.ConditionTrue)
 				checkOCMProgressing(operatorv1.ConditionFalse)
-				checkDSRolloutState(true)
-				checkDSRolloutState(false)
-				// there is no way programmatically to check leader election outcome
-				g.By("waiting 10s for controller-manager leader election to complete")
```

Comment:
Do not need to check for leader election - by the time rollout is complete all pods should be running and a leader should be set.